src/components/home/home.jsx

```diff
@@ -0,0 +1,32 @@
+import { useEffect, useState } from "react"
+import { getArticles } from "../../api"
+import ArticleCard from "../article-card/article-card"
+import "./home.css"
+
+function Home(){
+  const [articlesArray, setArticlesArray] = useState([])
+
+  useEffect(() => {
+    async function fetchArticles(){
```

Comment:
we don't have to wrap this in its own function, we can just call our getArticles function directly in the useEffect and set the articles array :)